src/request_builder.ts

```diff
@@ -125,6 +126,7 @@ export class RequestBuilder {
         ...(this.sort ? { _es_sort_fields: this.sort } : {}),
         ...this.params,
         ...(initialRequest ? {} : { size: 0, from: 0 }),
+        dictionary,
```

Comment:
this dictonary doesn't need to be included here, only needed at time of template setup